stable/jenkins/templates/jenkins-master-route.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.master.route.enabled }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+{{- if .Values.master.route.labels }}
```

Comment:
Should this line be here? If a user does not define labels, there would be no labels at all?

Reply:
Copy Paste issue, Fixed!

Reply:
Thanks!